asf_heat_pump_suitability/pipeline/transform/uprns.py

```diff
@@ -139,10 +147,10 @@ def parse_arguments() -> argparse.Namespace:
         uprns_gdf = uprns_gdf.sjoin(
             la_boundaries_gdf[["LAD23CD", "LAD23NM", "geometry"]],
             how="inner",
-            predicate="within",
+            predicate="intersects",
         ).drop(columns="index_right")
 
-    if args.scale.lower() == "plymouth_similar":
+    elif args.scale.lower() == "plymouth_similar":
```

Comment:
uff i missed this if that should've been an elif in the previous PR! good you found it.